test/kubernetes/e2e/tests/manifests/edge-gateway-test-helm.yaml

```diff
@@ -2,3 +2,21 @@ gateway:
   validation:
     # added to preserve behavior tested by this suite.
     warnMissingTlsSecret: false
+
+settings:
+  kubeResourceOverride:
+    spec:
+      observabilityOptions:
+        configStatusMetricLabels:
+          Gateway.v1.gateway.solo.io:
+            labelToPath:
+              name: '{.metadata.name}'
+              namespace: '{.metadata.namespace}'
+          Upstream.v1.gloo.solo.io:
+            labelToPath:
+              name: '{.metadata.name}'
+              namespace: '{.metadata.namespace}'
+          VirtualService.v1.gateway.solo.io:
+            labelToPath:
+              name: '{.metadata.name}'
+              namespace: '{.metadata.namespace}'
```

Comment:
Why is this added?

Reply:
The tests are run twice, once with the clearing and once without. This change ensures that non-clearing case is writing resource status metrics (it doesn't by default). tl;dr: The tests need it.